_posts/2020-03-24-dask-developer-workshop.markdown

```diff
@@ -0,0 +1,90 @@
+
+- Who are Dask users and what are their use cases?
+- What are current pain points and what needs to be fixed as soon as possible?
+- What is on the Dask user’s wish list?
+
+In other words, this was the chance for presenters to officially “rant” [*] about Dask, with experts having the experience and knowledge to help in the same room.
+This exchange between users and developers enabled immediate fixes of minor problems, identifying synergies between different projects, as well as shaping the roadmap for future development of Dask. We feel that this is a successful model for driving the Dask open-source community.
+
+[*] quote Matthew Rocklin
+
+## The Blue Yonder way of using Dask
+
+We at Blue Yonder are still finishing our migration from a proprietary job scheduler to Dask.distributed. This is why when _we_ talk about _Dask_ in this post, we are mainly referring to _Dask.distributed_, as this is our main use case.
+
+We quickly realized at the workshop that Blue Yonder’s use case is almost unique in the Dask community.
+
+Florian Jetter opened the stage at the conference and presented the typical data flow for our machine learning products, the usage of Dask and Dask.distributed within this flow, and where we were currently facing issues.
+ 
+At Blue Yonder, we provide Machine Learning as a Service to our customers in retail. The data which our customers provide through an API is inserted into a relational database. The Machine Learning and prediction steps need a denormalized data format, which we provide in form of Parquet datasets in the Azure Blob Store. The resulting predictions are written back to Parquet files and offered to the customer through an API. 
+The Data Engineering team leverages the distributed scheduler for parallel task execution during the data transformation steps between database and Parquet datasets. Most parts of these pipelines use Dask bags or delayed objects for map/reduce operations. Dask dataframes are especially useful when we reshuffle datasets with an existing partitioning to a differently partitioned dataset. We use [Kartothek](https://github.com/JDASoftwareGroup/kartothek) for this purpose.
+
+A lot of Dask users we met at the workshop also utilized Dask clusters for data-heavy calculations. However, the Blue Yonder use case remained special: most other users were working in an R&D-like environment. Their computations are not running in comparable production environments. Most importantly, these users had no service level agreements with customers about, for example, delivery times for predictions. It seems that Blue Yonder puts higher requirements on the stability and robustness of the distributed scheduler than any other of the users present at the conference.
+
+## Issues we have encountered with Dask and potential improvements
```

Comment:
The title doesn't summarize the content, IMHO.

```suggestion
## Issues with Dask that others have encountered too
```

Reply:
But with that suggestion we remove the notion that _we_ have encountered the issues also, is there a compromise?

Reply:
The "too" at the end implies that you've faced it. If you want to call it out more explicitly then maybe something like:
```suggestion
## We are not alone - Dask issues encountered by others too 
```
or
```suggestion
## Dask issues faced by us and others too

Reply:
@nefta-kanilmaz-jdas : _no_